provider/openstack/local_test.go

```diff
@@ -2680,6 +2680,22 @@ func (s *noNeutronSuite) TestUpdateGroupControllerNoNeutron(c *gc.C) {
 	))
 }
 
+func (s *noNeutronSuite) TestStartInstanceNetworkNotSetNoError(c *gc.C) {
```

Comment:
Eventually, we'd love to get rid of nova networking.  Only 1 or 2, I hope, sites still use it.  It was deprecated a couple of years ago.   Once that happens, we'll loose this test.  To get something like it working with the openstack testservice, we'd need to add the equivalent of s.Nova.SetAvailabilityZones(),  s.Neutron.SetNetworks() perhaps.  At that point SetExternalNetworks might be helpful as well.